python/tests/Makefile

```diff
@@ -17,4 +17,11 @@
 clean:
 	rm -rf build __pycache__ .pytest_cache *.so
 
-.PHONY: cln clean
+setup:
+	$(PYTHON) setup-qfunctions.py build
+
+TEST_OPTS ?= --ceed /cpu/self/ref/serial
+test: setup
```

Comment:
I'm fine with this because it's more foolproof. If we knew the name of the generated file, we could write a proper dependency for it instead of using a phony target.